ccd-definition/ComplexTypes/RepaymentPlanLRspec-CUI-nonprod.json

```diff
@@ -0,0 +1,26 @@
+[
+  {
+    "ID": "RepaymentPlanLRspec",
```

Comment:
and this one if it overrides existing file for lr

Reply:
Having a discussion with both and the change with be in base file as it is not impact the functionality. This file will be removed